### ISSUE TITLE
Can I flip dynamically generated views?

### ISSUE THREAD

**Issue by ticofab** [davideas/FlipView#10]
Hi, this is really just a question: is it possible to load dynamic views? Imagine an adapter that displays different views according to their position in the recyclerview. Can I flip those views, and have both front views and rear views dynamically generated? My idea is to have a flip view for each item and add dynamically both front and rear view but I am having trouble achieving it. Thanks for your time & attention.

**Comment by davideas** [comment]
Hello @ticofab, when firstly I did this library the purpose was exactly the yours.
Please have a look at the other my project [FlexibleAdapter](https://github.com/davideas/FlexibleAdapter) where in the [ExampleAdapter ](https://github.com/davideas/FlexibleAdapter/blob/master/flexible-adapter-app/src/main/java/eu/davidea/samples/flexibleadapter/ExampleAdapter.java) or in [SimpleItem ](https://github.com/davideas/FlexibleAdapter/blob/master/flexible-adapter-app/src/main/java/eu/davidea/samples/flexibleadapter/models/SimpleItem.java), I use the following code to handle flip and multiple selection together. I hope this is what you are looking for:

``` java
//ANIMATION EXAMPLE!! ImageView - Handle Flip Animation on Select ALL and Deselect ALL
if (adapter.isSelectAll() || adapter.isLastItemInActionMode()) {
    //Reset the flags with delay
    holder.itemView.postDelayed(new Runnable() {
        @Override
        public void run() {
            adapter.resetActionModeFlags();
        }
    }, 200L);
    //Consume the Animation
    holder.mFlipView.flip(adapter.isSelected(position), 200L);
} else {
    //Display the current flip status
    holder.mFlipView.flipSilently(adapter.isSelected(position));
}
```

Basically before activating the automatic flip based on the selection you should set the front/rear image with `setFrontImage()` and `setRearImage()` and/or backgrounds. Also pay attention of the click event of the FlipView in the ViewHolder: the tap should trigger the flip and activate the itemView as well. You can see this in the `SimpleItem`.
